config/crd/bases/mongodbcommunity.mongodb.com_mongodbcommunity.yaml

```diff
@@ -5,12 +5,6 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.4.1
-    service.binding/type: 'mongodb'
```

Comment:
We should find a way to keep this annotation when the pre-commit hook removes it.

Reply:
Should I revert it?

Reply:
Yes please